tests/commands/create/test_install_app_support_package.py

```diff
@@ -24,7 +24,7 @@ def test_install_app_support_package(create_command, myapp, tmp_path, support_pa
 
     # Confirm the right URL was used
     create_command.download_url.assert_called_with(
-        download_path=Path.home() / '.briefcase' / 'support',
+        download_path=create_command.dot_briefcase_path / 'support',
```

Comment:
This could be `download_path=tmp_path / 'dot-briefcase' / 'support'` instead. Would you prefer that?

Asking based on discussion at https://github.com/beeware/briefcase/pull/309 , which I skimmed.

Reply:
If we need to make a change for Windows, it's likely going to be a change of the ".briefcase" part, not the "support" part; so sticking with "dot_briefcase_path" for now will be fine.